controllers/rhmi/rhmi_controller.go

```diff
@@ -1499,3 +1618,34 @@ func getInstallation() (*rhmiv1alpha1.RHMI, error) {
 		},
 	}, nil
 }
+
+func formatAlerts(alerts []struct {
+	Labels models.LabelSet `json:"labels"`
+	State  string          `json:"state"`
+}) resources.AlertMetrics {
+	alertMetrics := resources.AlertMetrics{}
+
+	for _, alert := range alerts {
+		if alert.Labels["alertname"] == "DeadMansSwitch" {
```

Comment:
We decide to exclude the DMS for the reason it is always firing. Thinking about how this metric is going to be use. At the initial level all clusters rhoam_alerts are going to be be combined to to show how many alerts are firing across the fleet. It would be unfair to create a dashboard that would require to reader to calculate the number of clusters and taken that away from the number of firing alerts. 

What kind of issue do you think it can indicate? If there is a problem with monitoring the DMS services should page SRE. If for some reason the metric is not exported, the planned dashboards currently don't have any alerts planned. So it would be a SRE member we would be depending on to see the issue. If we are worried about the this metric getting turned off in a future update the better opinion would be a e2e test. Currently I don't think we test if any metrics are returned.

Reply:
Yeah that's a good point. The DMS not firing would page SRE so maybe there's no value in including it in the fleet wide metrics. We could filter it out of the dashboards but yeah that would be additional work for possibly no value

Reply:
Yap and then it seems easier to do this filtering in the Go code over in PromQL. Are you happy to leave this as it is now.